lib/presentation/mixins/send_files_mixin.dart

```diff
@@ -64,9 +64,9 @@ mixin SendFilesMixin {
     final result = await FilePicker.platform.pickFiles(
       withData: true,
     );
-    if (result == null && result?.files.isEmpty == true) return;
+    if (result == null || result.files.isEmpty) return;
 
-    sendFileOnWebInteractor.execute(room: room!, filePickerResult: result!);
+    sendFileOnWebInteractor.execute(room: room!, filePickerResult: result);
```

Comment:
Why do you change it, the old version is correct

Reply:
No it wasn't because, the room was created even if we cancel file picker, we can't have result == null and result empty at the same time

Reply:
okay you are correct 👍